protoc_docs/bin/py_docstring.py

```diff
@@ -42,18 +43,33 @@ def main(input_file=sys.stdin, output_file=sys.stdout):
     # Iterate over the data that came back and parse it into a single,
     # coherent CodeGeneratorResponse.
     answer = []
+    _BATCH_TOKEN = "CD985272F78311"
```

Comment:
Perhaps include newlines? It seems like two newlines on each side of this would make the mechanism safer.

Reply:
I tried many combinations with newlines here (before, after, before and after) and, unfortunately could not achieve a consistent result. It is probably because pandoc removes "unnecessary" newlines and reconstructing the result becomes unpredictable. 

I did verify this behavior on big files with lots of formatting, being able to achive 100% exact same result as the original implementation, so it is more or less reliable. Even if there is any issue, it should be comments-only and in the realm of extra/missing whitespace or something minor like that (i.e. should be still Ok).